package.json

```diff
@@ -24,14 +24,14 @@
     "html-webpack-plugin": "^3.2.0",
     "mocha": "^5.2.0",
     "mochapack": "^1.0.0",
-    "node-sass": "^4.12.0",
+    "node-sass": "^4.14.1",
     "sass-loader": "^7.1.0",
     "style-loader": "^0.23.1",
     "webpack": "^4.31.0",
-    "webpack-cli": "^3.2.1",
-    "webpack-dev-server": "^3.1.14"
+    "webpack-cli": "^3.3.12",
```

Comment:
Is the installation of webpack complete? the video made it seem a bit more complex

Reply:
Great question! And thanks for updating me on that video and more steps to take. 
Testing it now, it seems to work how we expect!